src/test/simplesigner_tests.cpp

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2019-2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <key.h>
+#include <pubkey.h>
+#include <script/interpreter.h>
+#include <script/sign.h>
+#include <script/signingprovider.h>
+#include <script/standard.h>
+#include <test/util/setup_common.h>
+#include <tinyformat.h>
+#include <uint256.h>
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(simplesigner_tests, BasicTestingSetup)
+
+BOOST_AUTO_TEST_CASE(simplesigner_base_tests)
+{
+    uint256 message;
```

Comment:
Is it intended that here (and in all following testcases) `message` is (implicitely) set to all-zero?

Reply:
It shouldn't really matter what the message is, but then there's that saying about bridges, so I'll set it to something, sure.